hooks/conftest.py

```diff
@@ -51,6 +51,26 @@ def user_filesystem(
     # create a filesystem with spm in a .cookiecutters directory and
     # template directories called article, other, and another
     # the article template contains a set of files defined in TEMPLATE_FILES
+    #
+    # directory structure:
+    # ├── .cookiecutters
+    # │   └── scikit-package-manuscript
+    # │       └── templates
+    # │           ├── another
+    # │           ├── other
+    # │           └── article
+    # │               ├── article.cls
```

Comment:
let's give article a better name too....`article-existing-in-skp-template.cls` or sthg